src/store/actions/todo.js

```diff
@@ -24,3 +25,10 @@ export const markTaskAsDone = (taskId) => {
     payload: taskId,
   };
 };
+
+export const editTaskFromTodo = (payload) => {
```

Comment:
if its a task then name it as task, or if it is an ID then name it as ID,
```suggestion
export const editTaskFromTodo = (taskId) => {
```

Reply:
parameter of `editTaskFromTodo` receives an object that does not contains all properties of `task` object. Thats why name it as `payload` to avoid confusions. @iud-wtag vai